src/lib/init.ts

```diff
@@ -523,3 +679,66 @@ function getPackageJSON(isJS: boolean) {
     },
   };
 }
+
+function getGatewayPackageJSON(isJS: boolean) {
+  return {
+    private: true,
+    name: "gateway",
+    description: "Fluence gateway for running aqua function via HTTP methods",
+    version: "0.0.1",
+    type: "module",
+    keywords: [],
+    scripts: {
+      ...(isJS
+        ? {}
+        : {
+            build: "rm -rf ./dist && tsc -p tsconfig.json",
+          }),
+      start: isJS
+        ? "node src/server.js"
+        : "node --loader ts-node/esm src/server.ts",
+    },
+    dependencies: {
+      [JS_CLIENT_NPM_DEPENDENCY]:
+        CLIPackageJSON.dependencies[JS_CLIENT_NPM_DEPENDENCY],
+      fastify: "4.25.2",
+      "@fastify/rate-limit": "9.1.0",
+      "@sinclair/typebox": "0.32.11",
+    },
+    devDependencies: isJS
+      ? {}
+      : {
+          "@types/node": "20.11.5",
+          "@fastify/type-provider-typebox": "4.0.0",
+          typescript: "5.3.3",
+          "ts-node": "10.9.2",
+        },
```

Comment:
also typescript can be updated in the frontend template
```suggestion
    ...(isJS
      ? {}
      : {
          devDependencies: {
            "@types/node": "20.11.5",
            "@fastify/type-provider-typebox": "4.0.0",
            typescript: "5.3.3",
            "ts-node": "10.9.2",
          },
        }),
```

Reply:
Didn't understand your code suggestion. There is no need to spread anything i believe. Or what's the reasoning behind this?

Reply:
It's just that if you don't have any `devDependencies` in js template there is no need to have an empty `devDependencies: {}` in package.json there. That's it

Reply:
Oh, so it's about empty `devDependencies`... Yeap, nice. Seems like i misread it the first time.